packages/node/src/core/evm/templates/template-verification.test.ts

```diff
@@ -0,0 +1,129 @@
+import { ethers } from 'ethers';
+import * as fixtures from 'test/fixtures';
+import * as verification from './template-verification';
+import { RequestErrorCode, RequestStatus } from 'src/types';
+
+describe('verify', () => {
+  const TEMPLATE_ID = '0x101c36202b92b358adda664515ea920f76b1edcf5f1285ab0c845f54638b45aa';
```

Comment:
For the verification tests, I used actual valid values, as this was necessary to actually validate templates. Most of the other unit tests in the app use invalid values like `apiCallId` or `templateId`. I'm still undecided which way is preferable.

Invalid values are much easier to read at a glance. Valid values are obviously closer to the real thing.

Reply:
I think only using valid values when you have to makes sense